tardis/model/geometry/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""
+Holding information about model geometry for radial 1D models.
```

Comment:
Here, again, it would be very helpful is you can say what this information includes as a second part of the docsting.

Reply:
Extended summary added. Please review again :)